layout/page.ftd

```diff
@@ -37,6 +37,7 @@ ftd.length.px spacing: 0
 page-widths widths: $widths
 ;; todo: is the follow reference to `.gap` correct?
 ds.gap margin: $ds.spaces.vertical-gap.large
+optional ftd.background background:
```

Comment:
Nopes, you are removing default background with this change. Make it: 

```suggestion
ftd.background.solid background: $ds.colors.background.base
```

Or something like this, so by default page gets a background.